typescript/src/bitcoin.ts

```diff
@@ -253,7 +247,7 @@ export interface Client {
 export function decomposeRawTransaction(
   rawTransaction: RawTransaction
 ): DecomposedRawTransaction {
-  const toHex = (bufferWriter: StaticWriter | BufferWriter) => {
+  const toHex = (bufferWriter: any) => {
```

Comment:
Non-blocking:

Based on the Typescript documentation [we should avoid using any](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#any). Just pointing that out since I don't have a solution for that as of now, so can't propose anything better 🤔